src/server/pfs/server/driver.go

```diff
@@ -90,11 +90,11 @@ type driver struct {
 	// and Authorization API
 	address      string
 	pachConnOnce sync.Once
-	onceErr      error
 	pachConn     *grpc.ClientConn
 
 	// pachClient is a cached Pachd client, that connects to Pachyderm's object
-	// store API and auth API
+	// store API and auth API. Instead of accessing it directly, functions should
+	// call getPachClient()
 	pachClient *client.APIClient
```

Comment:
Is there a convention we can supply here to further mark it private? Maybe name it `_pachClient` ?